### ISSUE TITLE
Doxygen creates call between functions if parameters start in new line (Origin: bugzilla #783579)

### ISSUE THREAD

**Issue by doxygen** [doxygen/doxygen#6216]
status NEW severity *normal* in component *general* for *---*
Reported in version *1.8.13* on platform *Other*
Assigned to: Dimitri van Heesch

Original attachment names and IDs:
- _[example.tgz](https://sourceforge.net/projects/doxygen/files/attachments/bug783579_353438_example.tgz/download/download)_ (ID 353438)

On 2017-06-09 07:17:15 +0000, Christoph Schmidt wrote:

    Created attachment 353438
    Example containing source code and Doxyfile to reproduce bug
    
    Declare two functions the following way:
    
    void f(void)
    {
    }
    
    void g
      (int p)
    {
    }
    
    Doxygen will create a call from f to g in the call graph as well as in the
    caller graph (tested with version 1.8.13).
    
    Please see also the attached example.

On 2017-06-09 07:21:01 +0000, Christoph Schmidt wrote:

    When removing the line break between function g and its parameter list, no
    call from f to g is created.

**Comment by jtchitty** [comment]
I'm also seeing this issue with Doxygen 1.8.13 with code written in the following style:
```
/**
 * Description of the function.
 *
 * @return true if blah
 */
static bool f
(
    int x ///< description of the parameter
)
{
    ...
}
```
We use this format, because it allows us to keep the parameter documentation extremely close to the parameters they describe, which increases the chances that the documentation will remain in-sync with the code.

But, the call graphs and reference lists are just wrong when we do this.

**Comment by jtchitty** [comment]
The following file demonstrates the issue.  When `CALL_GRAPH = YES`, it will show that `print_arg()` calls `main()`, which it clearly does not.
```
#include <stdio.h>
#include <stdlib.h>

/**
 * Print an argument on its own line.
 */
static void print_arg
(
    const char* arg ///< the argument to be printed.
)
{
    printf("%s\n", arg);
}

int main
(
    int argc, ///< Count of the number of command-line arguments.
    char **argv ///< Pointer to an array of null-terminated strings containing command-line args.
)
{
    printf("argc = %d\n", argc);
    for (int i = 0; i < argc; i++)
    {
        print_arg(argv[i]);
    }

    return EXIT_SUCCESS;
}
```

**Comment by jtchitty** [comment]
This appears to have been **fixed** in release **1.8.14**, and remains fixed in 1.8.20 and the current master branch.  This issue can be closed.
